go/noise/gaussian_noise_test.go

```diff
@@ -1052,21 +993,16 @@ func TestComputeConfidenceIntervalFloat64Gaussian(t *testing.T) {
 			epsilon:         0.05,
 			delta:           math.Inf(1), // delta should be strictly positive and smaller than 1.
 			alpha:           0.2,
-			want:            ConfidenceInterval{},
-			wantErr:         true,
 		},
 	} {
 		got, err := gauss.ComputeConfidenceIntervalFloat64(tc.noisedX, tc.l0Sensitivity, tc.lInfSensitivity, tc.epsilon, tc.delta, tc.alpha)
-		if (err != nil) != tc.wantErr {
-			t.Errorf("ComputeConfidenceIntervalFloat64: when %s got err %v, wantErr=%t", tc.desc, err, tc.wantErr)
-		}
-		if !approxEqual(got.LowerBound, tc.want.LowerBound) {
-			t.Errorf("TestComputeConfidenceIntervalFloat64(%f, %d, %f, %f, %f)=%0.10f, want %0.10f, desc %s, LowerBound is not equal",
-				tc.noisedX, tc.l0Sensitivity, tc.lInfSensitivity, tc.epsilon, tc.alpha, got.UpperBound, tc.want.UpperBound, tc.desc)
+		if err == nil {
+			t.Errorf("ComputeConfidenceIntervalFloat64Gaussian: when %s no error was returned, expected error", tc.desc)
 		}
-		if !approxEqual(got.UpperBound, tc.want.UpperBound) {
-			t.Errorf("TestComputeConfidenceIntervalFloat64(%f, %d, %f, %f, %f)=%0.10f, want %0.10f, desc %s, UpperBound is not equal",
-				tc.noisedX, tc.l0Sensitivity, tc.lInfSensitivity, tc.epsilon, tc.alpha, got.LowerBound, tc.want.LowerBound, tc.desc)
+		// Checks default confidence interval is returned when err is generated.
```

Comment:
we don't check this value. It can be easily changed. THe only thing we need to know is that there were an error.
Please fix in other places, if you have similar one.